tests/Routing/RouteParameterTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Illuminate\Tests\Routing;
+
+use PHPUnit\Framework\TestCase;
+use Illuminate\Routing\RouteParameter;
+
+class RouteParameterTest extends TestCase
+{
+    public function testParsingName()
+    {
+        $parameter = new RouteParameter('baz');
+        $this->assertEquals('baz', $parameter->name());
```

Comment:
I suppose al `assertEquals` comparing a string can be replace by stricter `assertSame` assertions ?